Framework/PythonInterface/plugins/algorithms/ComputeCalibrationCoefVan.py

```diff
@@ -0,0 +1,168 @@
+from mantid.api import PythonAlgorithm, AlgorithmFactory, WorkspaceProperty, Progress, InstrumentValidator
+from mantid.kernel import Direction
+import mantid.simpleapi as api
+import numpy as np
+from scipy import integrate
+import scipy as sp
+import mlzutils
```

Comment:
Not so great to see that we have to import *utilities* from a specific facility to get what looks like a general algorithm working, especially since `mlzutils.do_fit_gaussian` doesn't seem to really be facility specific. Or is it?